web_widget_child_selector/static/src/xml/widget_child_selector.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+    <t t-name="FieldChildSelector">
+
+        <!--<div class="o_element"/>-->
+        <t t-if="widget.mode === 'readonly'">
+            <a t-if="!widget.nodeOptions.no_open" t-att-tabindex="widget.attrs.tabindex" class="o_form_uri" href="#"/>
+            <span t-if="widget.nodeOptions.no_open"/>
+        </t>
+        <div t-if="widget.mode === 'edit'" class="o_field_widget">
+            <span class="o_input_value"/>
+            <div class="o_input_dropdown"/>
+        </div>
+    </t>
+    <t t-name="FieldChildSelectorChild">
+        <div class="row">
+            <div t-foreach="childs" t-as="key" class="col-xs-12">
+                <button t-att-data-id="childs[key][0]" t-att-data-index="key"
```

Comment:
Maybe a group of radio buttons would be nicer

Reply:
![imagen](https://user-images.githubusercontent.com/47532076/69534147-7f55a480-0f79-11ea-8261-cdc8d9ae2f85.png)